java/api/src/main/java/io/ray/api/utils/parallelactor/ParallelActor.java

```diff
@@ -0,0 +1,46 @@
+package io.ray.api.utils.parallelactor;
+
+import io.ray.api.ActorHandle;
+import io.ray.api.call.ActorTaskCaller;
+
+public class ParallelActor<A> implements ParallelActorCall<A> {
+
+  private ParallelStrategy strategy;
+  private int parallelNum;
+  private int lastIndex = -1;
+  private ActorHandle<? extends ParallelActorExecutor> parallelExecutorHandle = null;
+
+  public ParallelActor(ParallelStrategy strategy, int parallelNum, ActorHandle<? extends ParallelActorExecutor> handle) {
+    this.strategy = strategy;
+    this.parallelNum = parallelNum;
+    parallelExecutorHandle = handle;
+  }
+
+  public ParallelInstance<A> getParallel(int index) {
+    return null;
+  }
+
+  public ActorHandle<? extends ParallelActorExecutor> getParallelExecutorHandle() {
+    return parallelExecutorHandle;
+  }
+
+  public ParallelStrategy getStrategy() {
+    return strategy;
+  }
+
+  public int getParallelNum() {
+    return parallelNum;
+  }
+
+  public int getNextIndex() {
```

Comment:
Is this API needed? I think this API is not compatible with randomized strategies (if we will implement one later).

Reply:
If it's for internal use only, make drop `public`.

Reply:
And maybe the implementation should be in the runtime module?

Reply:
Yes. This API is moved to the strategy implementation.